alpha/engagements/2024/Rust Foundation/update-2024-07.md

```diff
@@ -0,0 +1,48 @@
+
+### Crate deletion RFC
+
+The crate deletion [RFC](https://github.com/rust-lang/rfcs/pull/3660) is now in FCP. Recall, Tobias wrote this RFC to provide crate owners with a mechanism to delete crates from crates.io under certain [conditions](https://github.com/Turbo87/rust-rfcs/blob/crates-io/crate-deletions/text/3660-crates-io-crate-deletions.md#proposal).
+
+### crates.io lib/bin detection
+
+Tobias backfilled `lib/bin` information for existing crate releases. The `has_lib` and `bin_names` columns in the crates.io database have been backfilled see [conversation](https://rust-lang.zulipchat.com/#narrow/stream/318791-t-crates-io/topic/lib.2Fbin.20detection)). For example, https://crates.io/crates/ripgrep finally shows `cargo install` now.
+
+### Crate specific RSS feed
+
+Thanks to Tobias, there is now a [crate-specific RSS feed](https://github.com/rust-lang/crates.io/pull/9064). You can subscribe to get all the details on the specific crates you are interested in. RSS feeds per crate are published at https://static.crates.io/rss/crates/{name}.xml. The feeds are synced with the database in a background job after every successful publish of the corresponding crate. It includes the latest 10 releases of the crate, but at least all releases within the last 24 hours.
+
+### Signing RFC Update
+
+Walter, along with other Rust Project members, are nearly complete in drafting the wording to convert the existing [Public Key Infrastructure (PKI) RFC](https://github.com/rust-lang/rfcs/pull/3579) to use TUF, which should alleviate many of the commented concerns.
+
+## Announcements and Community
+
+### New Infrastructure Engineer on Staff
+
+We are pleased to [announce](https://foundation.rust-lang.org/news/welcoming-infrastructure-engineer-marco-ieni-to-the-rust-foundation-team/) that Marco Ieni has joined the Rust Foundation as the newest member of our engineering team! Marco joins the Rust Foundation as its second Infrastructure Engineer, providing much-needed support to the Rust project and its infrastructure. As a member of the engineering team, Marco will ensure that the infrastructure supporting Rust and related projects is reliable, efficient, productive, and secure. He will be responsible for the operations of all current and future infrastructure, managing existing tools and new developments while working directly with the Rust Project Infrastructure team in their daily activities.
+
+### Sustainability Efforts
+
+Adam continues to focus on project sustainability, focusing on repo and maintainer information. Adam gave a Python specific version of his [Quantifying Nebraska talk](https://www.youtube.com/watch?v=QMHpy_mcx0Q) at [North Bay Python](https://pretalx.northbaypython.org/nbpy-2024/talk/9EXJ7T/). He is developing tooling around this.
+
+### Development of crates.io update
+
+Tobias [published](INSERT LINK HERE) an update of the development progress of crates.io.
```

Comment:
@Turbo87 Just waiting for the link on this one.

Reply:
yeah, unfortunately I still need a review approval from a team lead to be able to merge the blog post

Reply:
maybe we can link to the PR?